lib/models/payment.model.dart

```diff
@@ -32,7 +32,7 @@ enum PaymentType {
 extension PaymentTypeExtension on PaymentType {
   static const displays = {
     PaymentType.deposit: "Bitcoin Deposited",
-    PaymentType.withdraw: "Bitcoin Withdrawn",
+    PaymentType.sendOnChain: "Bitcoin Sent",
```

Comment:
IMHO `Withdrawn` was better, especially when he had the counterparty "Deposited".
If you really want to go for "Send", the other action should rather be "Receive"

Reply:
But surely when you are paying someone on-chain with your wallet you are not withdrawing coins, but rather sending them.

Reply:
1. most of the lightning wallets seem to limit the btc wallet functionality - up to a point of completely hiding it in some cases - it's only used to deposit/withdraw money, because send/receive should rather be made via lightning.
2. it's called 'withdraw' in ItchySats too, and also allows moving funds to any other arbitrary on-chain wallet.

what do you think about `Send/Receive`? (just like for lightning). one additional side-effect of the old naming was also the clear distinction between bitcoin/lightning actions.

Reply:
I'm not convinced anymore that 1) is a good idea. I'd like a wallet which can speak both, so that I can use funds on-chain if needed.

at 2) `withdraw` on ItchySats made sense because it was not designed to be a wallet. Here it is a wallet. Imho it should be `send/receive` here as well

Adding an additional point :)

3) I'm not a big fan of those labels in general, I think this can be designed better.

Reply:
`send/receive` works for me too! what irked me a bit was the `deposit/send` disjoint.

labels/icons can be easily changed, but the vocabulary of the UI matters :)

Reply:
I like this design: 
`You send, on-chain` vs `You sent, Lightning`

<img width="414" alt="image" src="https://user-images.githubusercontent.com/224613/202577694-de017a2c-3ce9-46b7-8edc-2a83ed224ba4.png">

Reply:
> I'm not convinced anymore that 1) is a good idea. I'd like a wallet which can speak both, so that I can use funds on-chain if needed.

this might as well be something that differentiates us a bit from other wallets; I like the explicitness of it and the additional useful feature.